### PR TITLE
Fixing dev build for dev16

### PR DESCRIPTION
**Pull request by jainaashish**
This is the minimum set of changes to fix dev build for dev16 developer command prompt. This PR removes VS 14.0 or 15.0 specific conditions from project files. There is another [wip] PR to remove all dev14 specific code which will also remove lot of source code conditioned on different VS versions.

### CODE REVIEW

**Comment by nkolev92** [comment]
Do you mind if we merge the dev14 PR before this one? 
It contains a lot more cleanup. 

I have fixed the issues, it just requires a 2nd review

**Comment by nkolev92** [comment]
Nevermind, I just noticed that you have approved it too. 
That one is merged :)

**Comment by nkolev92** [comment]
I think the majority of the changes you are making here are kcovered in the other PR. 

So rebasing should help minimize this change.

**Comment by jainaashish** [comment]
Glad atleast this PR inspired to merge that other PR which was long due :)

I'll rebase and update if more changes are needed to build dev with dev16

File: build/common.project.props [hunk 1]
```diff
@@ -86,15 +86,7 @@
...
-    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
-    <DefineConstants>$(DefineConstants);VS14</DefineConstants>
-    <VSSDKRoot>$(RepositoryRootDirectory)packages\Microsoft.VSSDK.BuildTools.14.3.25420</VSSDKRoot>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(VSSDKRoot)\tools</VSToolsPath>
-    <VsixOutputDirName>VS14</VsixOutputDirName>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(VisualStudioVersion)' == '15.0'">
+  <PropertyGroup>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
```
Comment (zivkan) [inline build/common.project.props]: NuGet.VisualStudio.Client has bumped MinimumVisualStudioVersion to 16.0. Should this one be bumped too?  Should NuGet.VisualStudio.Client.csproj use this value?

**Comment by jainaashish** [comment]
Ok rebased and this is the minimum set to build it with dev16 developer command prompt.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**